paimon-spark/paimon-spark-common/src/main/java/org/apache/paimon/spark/SparkGenericCatalog.java

```diff
@@ -284,6 +288,13 @@ private CaseInsensitiveStringMap autoFillConfigurations(
         Map<String, String> newOptions = new HashMap<>(options.asCaseSensitiveMap());
         fillAliyunConfigurations(newOptions, hadoopConf);
         fillCommonConfigurations(newOptions, sqlConf);
+
+        // add case-insensitive from sql conf
+        Option<String> insensitive = sqlConf.getAllConfs().get(sparkCaseInsensitive);
```

Comment:
`sqlConf.caseSensitiveAnalysis`?

Reply:
OK，sqlConf.caseSensitiveAnalysis is better.